features/old/accounts/invoices.feature

```diff
@@ -39,22 +39,24 @@ Feature: Show invoices from account's page (#16015909)
 
   Scenario: Display years of invoices when there are invoices
     Given a provider is logged in on 1st January 2011
-    Given a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
     And the provider is charging its buyers in prepaid mode
     And an invoice of buyer "zoidberg" for January, 2011
-    And an invoice of buyer "zoidberg" for January, 2012
-    And I go to the invoices issued by me
-    Then I should see the list of years with invoices have the following years:
-      | 0  |
+    And an invoice of buyer "zoidberg" for January, 2014 on 1st January 2014 (without jobs)
+    When I go to the invoices issued by me
+    Then the years selector should have the following options:
+      |      |
+      | 2014 |
```

Comment:
I've added two other options, just to validate what is going on exactly.

Having just two sequential years is straight forward. But what if there is a "gap" between the first and the last year? One can think that only those specific years will appear in the list - e.g. 2011 and 2014 in this case. But it's not what happens, in fact all years within the period are shown, whether or not they have any invoices.

Reply:
related: https://github.com/3scale/porta/pull/3485#discussion_r1288146054

Reply:
correct